internal/store/files/README.md

```diff
@@ -21,3 +21,24 @@ WHERE
   predicate = 'fips52AlphaCode'
   AND subject_id LIKE 'geoId/%'
 ```
+
+## recon_name_to_types.json generation
```

Comment:
nit: Intro line on what this file is, and why we need it?

Reply:
done